checker/src/main/java/org/checkerframework/checker/initialization/InitializationChecker.java

```diff
@@ -87,6 +88,20 @@ public InitializationChecker() {}
      */
     public abstract Class<? extends BaseTypeChecker> getTargetCheckerClass();
 
+    /**
+     * Also handle {@code AnnotatedFor} annotations for this checker. See {@link
+     * InitializationFieldAccessSubchecker#getUpstreamCheckerNames()} and the two implementations
```

Comment:
You also need this comment in the `InitializationFieldAccessSubchecker` version, otherwise someone could change that version without seeing the comment here.

Reply:
Oh, I see. Updated.